src/dryrun.ts

```diff
@@ -138,7 +138,7 @@ export async function createDryrun({
   await Promise.all(acctPromises);
 
   return new DryrunRequest({
-    txns: txns.map((st) => ({ ...st, txn: st.txn.get_obj_for_encoding()! })),
+    txns: txns.map((st) => ({ ...st, txn: st.txn.get_obj_for_encoding() })),
```

Comment:
no longer definitely not null/undefined?

Reply:
Prior to this change, the `Transaction.get_obj_for_encoding()` method could technically return `undefined` (only if the `Transaction` was a unknown type), so it was necessary to use the `!` operator in many places.

Now I've adjusted the return value so `undefined` is impossible. If the type is unknown (which shouldn't ever happen legitimately), it will throw an error